src/components/EditGrower.js

```diff
@@ -38,25 +38,40 @@ const EditGrower = (props) => {
   const [growerUpdate, setGrowerUpdate] = useState(null);
   const [loadingGrowerImages, setLoadingGrowerImages] = useState(false);
   const [saveInProgress, setSaveInProgress] = useState(false);
+  const [customUrl, setCustomUrl] = useState('nothing');
 
   useEffect(() => {
     async function loadGrowerImages() {
       if (grower?.id) {
         setLoadingGrowerImages(true);
         const selfies = await growerContext.getGrowerSelfies(grower.id);
         setLoadingGrowerImages(false);
-
+        const notAllowNull =
+          selfies !== undefined && selfies.includes(null)
+            ? selfies.filter((e) => e !== null)
+            : selfies;
         setGrowerImages([
-          ...(grower.imageUrl ? [grower.imageUrl] : []),
-          ...(selfies || []).filter((img) => img !== grower.imageUrl),
+          ...(grower.image_url ? [grower.image_url] : []),
+          ...(notAllowNull || []).filter((img) => img != grower.image_url),
+          customUrl === '' ? null : customUrl,
```

Comment:
I like this way of doing it, but we should put `customUrl` at the front of the array.